packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -23,17 +23,53 @@ import {
 	UsageError,
 	createChildMonitoringContext,
 } from "@fluidframework/telemetry-utils";
-import { IPendingContainerState } from "./container.js";
 import { ISerializableBlobContents, getBlobContentsFromTree } from "./containerStorageAdapter.js";
 
+export interface PendingSnapshot {
```

Comment:
i don't thing we should use pending in the name here, as it is used in cases besides pending. maybe just call it SnapshotWithBlobs